lms/djangoapps/verify_student/tests/test_signals.py

```diff
@@ -0,0 +1,51 @@
+"""
+Unit tests for the course_mode signals
+"""
+
+from datetime import datetime, timedelta
+
+from pytz import UTC
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from lms.djangoapps.verify_student.models import VerificationDeadline
+from lms.djangoapps.verify_student.signals import _listen_for_course_publish
+
+
+class CourseModeSignalTest(ModuleStoreTestCase):
+    """
+    Tests for the CourseMode model
+    """
+
+    def setUp(self):
+        super(CourseModeSignalTest, self).setUp()
+        self.end = datetime.now(tz=UTC).replace(microsecond=0) + timedelta(days=7)
+        self.course = CourseFactory.create(end=self.end)
+        VerificationDeadline.objects.all().delete()
+
+
+    def test_no_deadline(self):
+        """ Verify currency is always lowercase. """
+        _listen_for_course_publish('store', self.course.id)
+
+        self.assertIsNone(_listen_for_course_publish('store', self.course.id))
```

Comment:
`_listen_for_coures_publish` never returns a value, so I assume that we should be asserting on something else here?